doc/user_doc/vic_installation/introduction.md

```diff
@@ -1,50 +1,154 @@
-# Overview of vSphere Integrated Containers Engine for vSphere Administrators
+**VICe Overview for vSphere Admins**
```

Comment:
Or vSphere Integrated Containers Engine, to give it its full, legally sanctioned name :)